SYCL/ESIMD/accessor.cpp

```diff
@@ -26,8 +26,8 @@ constexpr unsigned int VL = 1024 * 128;
 using Ty = float;
 
 int main() {
-  Ty data0[VL] = {0};
-  Ty data1[VL] = {0};
+  Ty *data0 = new float[VL];
```

Comment:
the test is running only on Linux (see line 9).  Should we enable it on Windows as well?

Reply:
Yes. I will flip the switch once it will be fully enabled.